src/example_fgen_basic/error_v/creation.py

```diff
@@ -1,5 +1,5 @@
 """
-Wrappers of `m_error_v_creation` [TODO think about naming and x-referencing]
+Wrappers of [`m_error_v_creation`](/fortran-api/module/m_error_v_creation.html)
```

Comment:
oh very nice, I like it a lot

Reply:
```suggestion
Wrappers of [`m_error_v_creation`](../fortran-api/module/m_error_v_creation.html)
```

might need to be something like this, at least the current link doesn't work on https://example-fgen-basic--32.org.readthedocs.build/en/32/api/example_fgen_basic/error_v/creation/